RELEASE-NOTES.md

```diff
@@ -51,6 +51,8 @@ The Axom project release numbers follow [Semantic Versioning](http://semver.org/
 - Adds a new utility tool, `mesh_converter`, which converts between mesh formats. The first conversion
   is from a Pro-E tetrahedral mesh to an STL mesh of its boundary triangles.
 - Primal: Adds a method to determine if a point is contained within a Tetrahedron.
+- The `primal::BoundingBox` class' `expand()` and `shift()` methods were modified so they do
+  nothing when called on empty bounding boxes.
```

Comment:
I think "invalid" is clearer than "empty".

Reply:
Fine. invalid is technically more correct. I'll change it.